### ISSUE TITLE
Superset install pulls in werkzeug 1.0 which has changed paths for imports, breaking the intsall

### ISSUE THREAD

**Issue by johnseekins** [apache/incubator-superset#9110]
### Expected results
Superset runs

### Actual results
```root@use1-hadoop-16:/datto/monitoring/superset# /datto/monitoring/superset/.pyenv/versions/3.6.8/bin/superset init
Traceback (most recent call last):
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/bin/superset", line 21, in <module>
    from superset.cli import create_app
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/superset/__init__.py", line 27, in <module>
    from flask_appbuilder import AppBuilder, IndexView, SQLA
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/flask_appbuilder/__init__.py", line 5, in <module>
    from .api import ModelRestApi  # noqa: F401
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/flask_appbuilder/api/__init__.py", line 10, in <module>
    from flask_babel import lazy_gettext as _
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/flask_babel/__init__.py", line 19, in <module>
    from werkzeug import ImmutableDict
ImportError: cannot import name 'ImmutableDict'
```
root@use1-hadoop-16:/datto/monitoring/superset# /datto/monitoring/superset/.pyenv/versions/3.6.8/bin/superset init
Traceback (most recent call last):
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/bin/superset", line 21, in <module>
    from superset.cli import create_app
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/superset/__init__.py", line 27, in <module>
    from flask_appbuilder import AppBuilder, IndexView, SQLA
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/flask_appbuilder/__init__.py", line 5, in <module>
    from .api import ModelRestApi  # noqa: F401
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/flask_appbuilder/api/__init__.py", line 10, in <module>
    from flask_babel import lazy_gettext as _
  File "/datto/monitoring/superset/.pyenv/versions/3.6.8/lib/python3.6/site-packages/flask_babel/__init__.py", line 19, in <module>
    from werkzeug import ImmutableDict
ImportError: cannot import name 'ImmutableDict'

#### How to reproduce the bug
1. Install superset per regular instructions: `pip3 install apache-superset`
2. attempt to run a superset command

### Environment

- superset version: 0.35.2
- python version: 3.6.8
- node.js version: ? (not installed as this is a binary install)
- npm version: ? (not installed as this is a binary install)

### Checklist

Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by johnseekins** [comment]
```
pip3 uninstall werkzeug
pip3 install werkzeug==0.16.1
```
Fixed the problem.

**Comment by nytai** [comment]
supersets requirements specifies `werkzeug==0.16.1` https://github.com/apache/incubator-superset/blob/master/requirements.txt#L82

This issue appears related to your environment.

**Comment by johnseekins** [comment]
It must be, yeah. Strange that just `pip3 install apache-superset` wouldn't pull the correct requirements.txt. Well...never mind then!

**Comment by villebro** [comment]
Werkzeug actually isn't restricted in `setup.py`, which means that `pip install apache-superset` pulls in the latest stable version. I noticed that airflow hit a similar problem with werkzeug this weekend, so superset also needs to either restrict the version or accommodate to the `1.0` version. I can look i to fixing this if needed.

**Comment by johnseekins** [comment]
I mean...docs indicate that `pip3 install apache-airflow` is the correct way to install latest stable, so it may be worth fixing.

**Comment by villebro** [comment]
@dpgaspar looking more closely at the stacktrace, it seems FAB is pulling in `Flask-Babel<1.0`, which pulls in an unpinned version of `werkzeug` but actually depends on `werkzeug<1.0`. As this has been fixed in `Flask-Babel==1.0.0` which was released last week, I think the cleanest solution would be bump `werkzeug>=1.0.0` in FAB if it doesn't introduce any other problems.

**Comment by dpgaspar** [comment]
Yes, totally agree @villebro, I'll do that test and release

**Comment by dpgaspar** [comment]
Fixed with #9117
